salt/metalk8s/volumes/prepared/files/metalk8s-volume-clean-up

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
```

Comment:
We should specified 2 or 3 (seeing the import future I guess it's 2, but better be explicit).

Reply:
And somewhere make sure it's installed on the system. Sure, we can assume it is (we're running `salt-minion` after all), but still...

Reply:
Sticking with Python 2 for now, not exactly sure where we should check if it's installed though. In the service template? Or through Salt?

Reply:
Salt. This is where maybe at some point these files shouldn't be dropped as-is by Salt, but packaged properly, which then allows to add a dependency on some Python version.